lasagne/init.py

```diff
@@ -50,6 +106,45 @@ def sample(self, shape):
 
 
 class Glorot(Initializer):
+    """Glorot weight initialization [1]_
+
+    This is also known as Xavier initialization.
+
+    Parameters
+    ----------
+    initializer : lasagne.init.Initializer
+        Initializer used to sample the weights. The default is Gaussian.
```

Comment:
Suggestion:
Initializer used to sample the weights, must accept `std` in its constructor to sample from a distribution with a given standard deviation. Defaults to lasagne.init.Normal.

@benanne: Shouldn't the default be `Uniform` because that's what they do in the paper? Or shouldn't it have a default at all?